File: 2351-first-letter-to-appear-twice/2351-first-letter-to-appear-twice.py
class Solution(object):
    def repeatedCharacter(self, s):
        c=''
        min=len(s)
        for i in s:
            index=s.find(i,s.find(i)+1)
            if index < min and index!=-1:
                min=index
                c=i
        return c
        

File: 2351-first-letter-to-appear-twice/2351-first-letter-to-appear-twice.java
class Solution {
    public char repeatedCharacter(String s) {
        char c=' ';
        int min=s.length();
        int n=0;
        for(int i=0;i<s.length();i++){
            n=s.indexOf(s.charAt(i),i+1);
            if(min>n && n!=-1){
                min=n;
                c=s.charAt(i);
            }
        }
        return c;
    }
}